.github/workflows/stale.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues
+
+on:
+  #pull_request:
+  
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: windows-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        ascending: true # older issues first
+        stale-issue-label: 'no-issue-activity'
+        stale-issue-message: 'This issue is stale because it has been open 180 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+        days-before-stale: 180
+        days-before-pr-stale: -1 # issues only
+        days-before-close: 5
+        # any-of-labels: documentation
+        only-labels: code-vs-markup-mismatch
+        exempt-issue-labels: 'feature proposal'
+        exempt-all-milestones: true
+        debug-only: ${{ github.event_name == 'pull_request' }} # Dry-run changes to the workflow
```

Comment:
what does this mean? only CI and release pipelines would execute?

Reply:
no, the 'on' is only allowing pull request and manual. I commented it out to be 100% sure that it doesn't run on every pull request change in the repo.